orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/expressions/ExpressionsSupport.java

```diff
@@ -367,6 +370,66 @@ static String judgment(Object obj, String id) {
     throw new IllegalArgumentException("An execution is required for this function");
   }
 
+  /**
+   * Gets value of given label key in manifest of given kind deployed by stage of given name
+   * @param obj #root.execution
+   * @param stageName the name of the stage to find
```

Comment:
From reading the source, looks like this is "the name of a `deployManifest` stage to find", right?